.reek.yml

```diff
@@ -22,7 +22,7 @@ detectors:
     max_calls: 1
     allow_calls: []
   FeatureEnvy:
-    enabled: true
+    enabled: false
```

Comment:
I think it would be better to exclude the exact example, instead of skipping the whole rule